Application/Transfer/Import/FSE/Service.php

```diff
@@ -0,0 +1,1116 @@
+                                            $remark = '';
+                                        }
+
+                                        Club::useService()->insertMeta(
+                                            $tblPerson,
+                                            '',
+                                            $entryDate,
+                                            '',
+                                            $remark
+                                        );
+                                    }
+
+                                    // Contact
+                                    $importService->insertPrivatePhone($tblPerson, 'Tel', $RunY);
+                                    $importService->insertPrivatePhone($tblPerson, 'mobil', $RunY);
+                                    $importService->insertPrivateFax($tblPerson, 'Fax', $RunY);
+                                    $importService->insertPrivateMail($tblPerson, 'email', $RunY);
+
+                                    $groupName = trim($Document->getValue($Document->getCell($Location['Stat_text'], $RunY)));
+                                    if ($groupName !== '') {
+                                        if (($tblGroup = Group::useService()->insertGroup(
+                                            $groupName
+                                        ))) {
+                                            Group::useService()->addGroupPerson(
+                                                $tblGroup,
+                                                $tblPerson
+                                            );
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    return
+                        new Success('Es wurden ' . $countMembers . ' Personen erfolgreich angelegt.')
+                        . new Success('Es wurden ' . $countCompanies . ' Firmen erfolgreich angelegt.')
+                        . new Layout(new LayoutGroup(new LayoutRow(new LayoutColumn(
+                            new Panel(
+                                'Fehler',
+                                $error,
+                                Panel::PANEL_TYPE_DANGER
+                            )
+                        ))));
+
+                } else {
+                    return new Warning(json_encode($Location)) . new Danger(
+                            "File konnte nicht importiert werden, da nicht alle erforderlichen Spalten gefunden wurden");
+                }
+            }
+        }
+
+        return new Danger('File nicht gefunden');
+    }
+
+    /**
+     * @param IFormInterface|null $Form
+     * @param UploadedFile $File
+     *
+     * @return IFormInterface|Danger|string
+     */
+    public function createCompaniesFromFile(IFormInterface $Form = null, UploadedFile $File = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Form" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)